codalab/rest/interpret.py

```diff
@@ -680,6 +680,10 @@ def interpret_file_genpath(target_cache, bundle_uuid, genpath, post):
                             info = ''.join(contents)
         except NotFoundError:
             pass
+        except PermissionError:
+            # Use the array of length 3 to pass the PermissionError to the frontend
+            info = ["Forbidden", "Forbidden", "Forbidden"]
```

Comment:
Would it be easy to update the worksheet interpret unit tests to cover this PermissionError? If it's not too difficult, let's try to add a test case.